tests/upgrades/test_virtwho.py

```diff
@@ -90,6 +91,9 @@ def test_pre_create_virt_who_configuration(self):
             3. Report is sent to satellite.
             4. Virtual sku can be generated and attached.
         """
+        default_loc_id = entities.Location().search(
+            query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
+        entities.Location(id=default_loc_id).read()
```

Comment:
Please use below way to use `default_location`
```
default_loc_id =  entities.Location().search(query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
default_loc = entities.Location(id=default_loc_id).read()
org = entities.Organization(name=self.org_name).create() 
default_loc.organization.append(entities.Organization(id=org.id))
default_loc.update(['organization'])
```

Reply:
Once you done with it than give try to run test as well ..

Reply:
Thanks @vijay8451  modified. Test result soon